docs/docs/workflows/create.md

```diff
@@ -71,12 +71,44 @@ Create the `constellation-state.yaml` using the output from the Terraform state
 ```bash
 CONSTELL_IP=$(terraform output ip)
 CONSTELL_INIT_SECRET=$(terraform output initSecret | jq -r | tr -d '\n' | base64)
-touch constellation-state.yaml
-yq eval '.version ="v1"' --inplace constellation-state.yaml
 yq eval '.infrastructure.initSecret ="$CONSTELL_INIT_SECRET"' --inplace constellation-state.yaml
 yq eval '.infrastructure.clusterEndpoint ="$CONSTELL_IP"' --inplace constellation-state.yaml
 ```
 
+</tabItem>
+<tabItem value="self-managed" label="Self-Managed">
+
+Self-managed infrastructure allows for managing the cloud resources necessary for a Constellation cluster separately from the Constellation CLI and Terraform,
```

Comment:
```suggestion
Self-managed infrastructure allows for managing the cloud resources necessary for a Constellation cluster separately from the Constellation CLI,
```
Our default workflow does not directly include Terraform, but a user might wish to use Terraform to self-manage the resources.